lib/chai/interface/assert.js

```diff
@@ -2860,9 +2859,8 @@ module.exports = function (chai, util) {
 
   assert.decreasesButNotBy = function (fn, obj, prop, delta, msg) {
     if (arguments.length === 4 && typeof obj === 'function') {
-      var tmpMsg = delta;
-      delta = prop;
-      msg = tmpMsg;
+      msg = 'getter function';
+      prop= null;
```

Comment:
Same here, if we have four arguments to this I think that `prop` must not be `null`.

Also, a space is missing between `prop` and the `=` sign.

Reply:
OMG, he notice :smile: , indeed that's my mistake. Did we need [eslint](https://eslint.org/)?